src/main.py

```diff
@@ -76,26 +76,7 @@ def main() -> None:
     for option in env.options:
         option.params_space.seed(CFG.seed)
     assert env.goal_predicates.issubset(env.predicates)
```

Comment:
@tomsilver This refactor moves this code to utils.py, since dataset `__init__.py` needs it too. Should we refactor this change into a separate PR? (Added benefit: test_main.py will get a nice speedup bc we don't need to run the whole pipeline just to test predicate exclusion lol)

Reply:
sure sounds good to me

Reply:
#380